Formula/gmp.rb

```diff
@@ -17,7 +17,7 @@ class Gmp < Formula
   def install
     # Enable --with-pic to avoid linking issues with the static library
     args = %W[--prefix=#{prefix} --enable-cxx --with-pic]
-    args << "--build=core2-apple-darwin#{`uname -r`.to_i}" if build.bottle?
```

Comment:
We can replace `core2` by `nehalem` on Mojave, if we want: it is accepted by the build, and will use specific asm.

Reply:
@fxcoudert Good idea. Will hold off until we figure out some sort of better DSL or similar.